chapters/ch35.md

```diff
@@ -1,43 +1,35 @@
 
---> _replace THIS LINE by your translation for the above line_
-
+Bạn đang làm về nhận dạng giọng nói và bạn có một tập dữ liệu âm thanh. Giả sử, tập dữ liệu của bạn có nhiều âm thanh nhiễu mà thậm chí con người còn mắc phải 10% lỗi. Giả sử, hệ thống của bạn đã đạt được 8% lỗi. Bạn có thể sử dụng bất kỳ kỹ thuật nào trong ba kỹ thuật được mô tả trong Chương 33 để tiếp tục tiến bộ nhanh chóng?
 
 > If you can identify a subset of data in which humans significantly surpass your system, then you can still use those techniques to drive rapid progress. For example, suppose your system is much better than people at recognizing speech in noisy audio, but humans are still better at transcribing very rapidly spoken speech.
 
---> _replace THIS LINE by your translation for the above line_
-
+Nếu bạn có thể xác định một tập dữ liệu con trong đó con người vượt qua được hệ thống của bạn một cách đáng kể, thì bạn vẫn có thể sử dụng các kỹ thuật đó để thúc đẩy tiến trình nhanh chóng. Ví dụ, giả sử hệ thống của bạn tốt hơn nhiều so với con người trong việc nhận dạng giọng nói trong âm thanh nhiễu, nhưng con người vẫn làm tốt hơn trong việc phiên âm lời nói rất nhanh.
 
 > For the subset of data with rapidly spoken speech:
 
---> _replace THIS LINE by your translation for the above line_
-
+Đối với tập dữ liệu con với lời nói nhanh:
 
 > 1. You can still obtain transcripts from humans that are higher quality than your algorithm’s output.
 
---> _replace THIS LINE by your translation for the above line_
-
+1. Bạn vẫn có thể nhận được bảng ghi từ con người với chất lượng cao hơn so với đầu ra thuật toán của bạn.
 
 > 2. You can draw on human intuition to understand why they correctly heard a rapidly spoken utterance when your system didn’t.
 
---> _replace THIS LINE by your translation for the above line_
-
+1. Bạn có thể dựa vào trực giác để hiểu lý do tại sao họ nghe chính xác một phát ngôn nhanh khi hệ thống của bạn không thành công.
 
 > 3. You can use human-level performance on rapidly spoken speech as a desired performance target.
 
---> _replace THIS LINE by your translation for the above line_
-
+1. Bạn có thể dùng chất lượng ở cấp độ con người trên lời nói nhanh như một mục tiêu với chất lượng mong muốn.
 
 > More generally, so long as there are dev set examples where humans are right and your algorithm is wrong, then many of the techniques described earlier will apply. This is true even if, averaged over the entire dev/test set, your performance is already surpassing human-level performance.
 
---> _replace THIS LINE by your translation for the above line_
-
+Tổng quát hơn, miễn là có các mẫu về tập phát triển mà con người đúng và thuật toán của bạn sai, thì nhiều kỹ thuật được mô tả trước đây sẽ áp dụng được. Điều này đúng ngay cả khi, tính trung bình trên toàn bộ tập phát triển/kiểm tra, chất lượng của bạn đã vượt qua chất lượng ở cấp độ con người.
```

Comment:
- "về" --> trong 
- thêm từ "rất" cho khớp nghĩa "many"
- Điều này đúng ngay cả khi chất lượng của bạn (tính trung bình trên toàn bộ tập phát triển/kiểm tra) đã vượt qua chất lượng ở cấp độ con người.